Multisig/Logic/Models/PrivateKey/PrivateKey.swift

```diff
@@ -19,26 +20,36 @@ struct PrivateKey {
         Address(_store.address)
     }
 
-    var data: Data {
+    struct PKData: Codable {
+        let key: Data
+        let mnemonic: String?
+    }
+
+    /// needed for PrivateKeyController.migrateLegacySigningKey()
+    var keyData: Data {
```

Comment:
maybe calling it `legacyKeyData` would be better

Reply:
ok